bin/main.rs

```diff
@@ -137,6 +137,11 @@ struct Args {
     )]
     api_prefix: String,
 
+    /// The Stream topic prefix to use. This is an advanced setting that should only be used if you
```

Comment:
Can you add just a bit more documentation here? Something that says this sets the prefix for all the internal streams so you can run different wadms within the same environment?

Reply:
yep. no problem. I updated the comment to reflect the intended use based on the [#180](https://github.com/wasmCloud/wadm/issues/180) description. let me know if it needs some improvement.

Reply:
Nope, looks good. Thank you!